src/mlstacks/terraform/gcp-remote-state/outputs.tf

```diff
@@ -1,3 +1,3 @@
-output "bucket_url" {
-  value = "gs://${google_storage_bucket.terraform_state.name}"
+output "remote_state_bucket_url" {
+  value = module.gcp-remote-state.bucket_url
```

Comment:
same here, where is `bucket_url` defined?

Reply:
Defined in module. See above.